docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -357,9 +357,7 @@ $ journalctl -fu dm-event.service
 If you run into repeated problems with thin pool, you can use the
 `dm.min_free_space` option to tune the Engine behavior. This value ensures that
 operations fail with a warning when the free space is at or near the minimum.
-For information, see <a
-href="../../../reference/commandline/dockerd/#storage-driver-options"
-target="_blank">the storage driver options in the Engine daemon reference</a>.
+For information, see [the storage driver options in the Engine daemon reference](../../reference/commandline/dockerd.md#storage-driver-options).
```

Comment:
Thanks @thaJeztah. I checked the rendered web page and it seems this link is OK. The pull request has been updated so that this link is reverted back to the original.

Now only the other link that caused the Jenkins error is changed. Please let me know if there are any other issues.